src/ioctls/system.rs

```diff
@@ -595,6 +600,14 @@ mod tests {
         assert!(cpuid_entries.len() <= KVM_MAX_CPUID_ENTRIES);
     }
 
+    #[cfg(any(target_arch = "x86", target_arch = "x86_64"))]
+    #[test]
+    fn test_get_supported_cpuid_fail_nentries_high() {
```

Comment:
nit: this is small enough to be another check in the `test_get_supported_cpuid()` test instead of a standalone test

Reply:
Okay will do it there.